litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/ProbeDetails.tsx

```diff
@@ -15,7 +15,12 @@ const ProbeDetails: React.FC<ProbeDetailsProps> = ({
 }) => {
   const classes = useStyles();
   const { t } = useTranslation();
-
+  const [httpData, setHttpData] = useState<any>({
```

Comment:
Can you please type this!

Reply:
Sure, have made the changes. PTAL @S-ayanide